htdocs/web_portal/GOCDB_monitor/ops_monitor_check.php

```diff
@@ -1,49 +1,22 @@
 <?php
 require_once "tests.php";
 
-// function returns associative array
-$res[1] = test_db_connection();
-
-$res[2] = test_url(Factory::getConfigService()->GetPiUrl().get_testPiMethod());
-
-$res[3] = test_url(Factory::getConfigService()->GetPortalURL());
-
-$counts=array(
-        "ok" => 0,
-        "warn" => 0,
-        "error" => 0
-);
-
-foreach ($res as $r){
-    $counts[$r["status"]]++;
-}
-/*
-If there is an error, counts["error"] is incremented, e.g.
-Array
-(
-    [ok] => 0
-    [warn] => 0
-    [error] => 1
-)
- */
-
-
 /* If someone wants to test the failure, they can fake one using
  * the fake_failure parameter */
-if(isset($_REQUEST['fake_failure'])) {
-    header("HTTP/1.0 500");
-    echo("GOCDB Web Portal is unable to connect to the GOCDB back end database\n");
-    die();
+if (isset($_REQUEST['fake_failure'])) {
+    $message = 'Fake failure';
+    $errorCount = 1;
+} else {
+    $errorCount = run_tests($message);
 }
 
-if ($counts["error"] != 0) {
+if ($errorCount != 0) {
```

Comment:
How does `$errorCount` get populated? In `check.php`, there is this line

```
$errorCount = run_tests($message);
```

 `ops_monitor_check.php` doesn't seem to call that method.

Reply:
ah you're right, didn't see that bit. Which perhaps suggests some additional comment is needed.

Maybe something like

```
/* If someone wants to test the failure, they can fake one using
 * the fake_failure parameter. 
 * Otherwise, run the tests. */
```

Reply:
There is already 'something like' that in the header which seems to me a better place.